src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,144 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.InvalidUserDataException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @Test
+    public void register_validUser_successfulRegistration_OK() {
+        user.setLogin("testUser");
+        user.setPassword("testPassword");
+        user.setAge(20);
+        assertDoesNotThrow(() -> registrationService.register(user));
+    }
+
+    @Test
+    public void register_minimumValidAge_Ok() {
+        user.setLogin("testUserMinAge");
+        user.setPassword("testPassword");
+        user.setAge(18);
+        assertDoesNotThrow(() -> registrationService.register(user),
+                "Registration using minimum valid age should be successful");
```

Comment:
It's reraly required to use assertDoesNotThrow
If logic in test throws exception that is not expected with assertThrows then test is automatically fail. What is same as using assertDoesNotThrow
```suggestion
        registrationService.register(user);
```